flow360/component/simulation/unit_system.py

```diff
@@ -917,6 +924,13 @@ class Flow360LengthUnit(_Flow360BaseUnit):
     unit_name = "flow360_length_unit"
 
 
+class Flow360AngleUnit(_Flow360BaseUnit):
+    """:class: Flow360AngleUnit"""
+
+    dimension_type = AngleType
+    unit_name = "flow360_angle_unit"
```

Comment:
How does the conversion to Flow360UnitSystem on AngleType work? Will I always get for example angles in radian regardless of what the user input is (rad/deg) once the conversion to Flow360UnitSystem is done?

Reply:
Entry for conversion system was missing - will fix.

Reply:
Fixed, entry in conversion system added.